src/repos/MatchRepo.ts

```diff
@@ -0,0 +1,51 @@
+import { getConnectionManager, Repository } from 'typeorm';
+import Constants from '../common/constants';
+import { MatchUpdateFields } from '../common/types';
+import Match from '../entities/Match';
+import User from '../entities/User';
+import UserRepo from './UserRepo';
+import AppDevUtils from '../utils/AppDevUtils';
+
+const db = (): Repository<Match> => getConnectionManager().get().getRepository(Match);
+
+const createMatch = async (users: User[]): Promise<boolean> => {
+  if (users.length < 2) return false;
+  const match = db().create({
+    users,
+    status: Constants.MATCH_CREATED,
+  });
+  await db().save(match);
+  return true;
+};
+
+const getActiveMatchesByNetID = async (netID: string): Promise<Match[]> => {
```

Comment:
getWeeklyMatchesByNetID

Reply:
oo I saw the change in your PR so I avoided it, but I can update this one!